pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -153,8 +159,19 @@ void updateSegmentMetrics() {
     HelixAdmin helixAdmin = _pinotHelixResourceManager.getHelixAdmin();
     int realTimeTableCount = 0;
     int offlineTableCount = 0;
+    int disabledTableCount = 0;
     ZkHelixPropertyStore<ZNRecord> propertyStore= _pinotHelixResourceManager.getPropertyStore();
 
+    // check if we need to log disabled tables log messages
+    boolean logDisabledTables = false;
+    long now = System.currentTimeMillis();
+    if (now -_lastDisabledTableLogTimestamp >= DISABLED_TABLE_LOG_INTERVAL_MS) {
+      logDisabledTables = true;
```

Comment:
would it make sense to name this variable as 'noisy' and use it for other logs/warnings as well? Not sure what else is there in this class.

Reply:
On a quick review, I didn't find others that might be candidates and didn't want to change any existing logs that we might be relying on. If we have more such instances then we could update the variable and logic accordingly at that time.